sympy/functions/elementary/tests/test_hyperbolic.py

```diff
@@ -1540,3 +1544,10 @@ def test_issue_25847():
     #acsch
     assert acsch(sin(x)/x).as_leading_term(x) == log(1 + sqrt(2))
     raises(PoleError, lambda: acsch(exp(1/x)).as_leading_term(x))
+
+
+def test_issue_25175():
+    x = Symbol('x')
+    g1 = 2*acosh(1+2*x/3)-acosh(S(5)/3-S(8)/3/(x+4))
+    g2 = 2*log(sqrt((x+4)/3)*(sqrt(x+3)+sqrt(x))**2/(2*sqrt(x+3)+sqrt(x)))
```

Comment:
Maybe take care of spacing here. I would write the following instead. 

`g1 = 2*acosh(1 + 2*x/3) - acosh(S(5)/3 - S(8)/3/(x + 4))`

Reply:
Update g2 accordingly :)

Reply:
Oops! changed them.